src/Consolonia.Core/Helpers/Observable.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using Avalonia.Reactive;
+
+namespace Consolonia.Core.Helpers
+{
+    /// <summary>
+    ///     Provides common observable methods as a replacement for the Rx framework.
+    /// </summary>
+    /// <remarks>
+    ///     COPIED FROM AVALONIA.BASE BECAUSE IT IS NOT PUBLICLY ACCESSIBLE,
```

Comment:
agreed, I wasn't including it because this is what Avalonia was doing. I just couldn't figure out how to do the binding like you showed me, that removes the need for this entirely. Thanks,